src/sonic_ax_impl/mibs/ietf/rfc1213.py

```diff
@@ -264,6 +298,29 @@ def _get_counter(self, oid, table_name):
             mibs.logger.warning("SyncD 'COUNTERS_DB' missing attribute '{}'.".format(e))
             return None
 
+    def aggregate_counters(self):
```

Comment:
>aggregate_counters [](start = 8, length = 18)

Is it only for Mellanox SKUs?

Reply:
This will only be executed if COUNTERS DB contains the rif counters. If RIF counters is not supported/disabled we will have unchanged behavior.